reclass/datatypes/parameters.py

```diff
@@ -115,7 +116,7 @@ def _extend_list(self, cur, new, path):
             ret = [cur]
             offset = 1
 
-        for i in xrange(len(new)):
+        for i in range(len(new)):
```

Comment:
Much like with items, maybe better to do this with builtings or past than by changing the behavior of the code in python2